### PR TITLE
Update documentation about pyls

### PR DESCRIPTION
**Pull request by ykqmain**
Added a configuration example about pyls.

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -757,7 +757,59 @@ If you use a virtualenv for your current project, add a path to it in your [proj
...
+	{
+        "pyls": {
+            "enabled": true,
+            "command": ["pyls"],
+            "scopes": ["source.python"],
+            "syntaxes": ["Packages/Python/Python.sublime-syntax", "Packages/MagicPython/grammars/MagicPython.tmLanguage", "Packages/Djaneiro/Syntaxes/Python Django.tmLanguage"],
+            "languageId": "python",
+            "settings": {
+                "pyls": {
+                    "configurationSources": ["flake8"],
```
Comment (rchl) [inline docs/index.md:776]: Maybe also include "pycodestyle" source.
Or all possible options but commented out.

File: docs/index.md [hunk 2]
```diff
@@ -757,7 +757,59 @@ If you use a virtualenv for your current project, add a path to it in your [proj
...
+                            "enabled": true,  // Turn on automatic completion by Jedi
+                            "extra_paths": [
+                                // "The directory where the pip installation package is located",
+                            ],
+                        },
+                        "pyflakes": {"enabled": true,},
+                        "pycodestyle": {
+                            "enabled": true,
+                            "ignore": [
+                                "E501",   // ignore lint
```
Comment (rchl) [inline docs/index.md:788]: I'm not sure we should be included opinionated rules in suggested config.
I would at least comment it out. And replace current comment with `Line too long (82 > 79 characters)` as the current one is not helpful.

File: docs/index.md [hunk 3]
```diff
@@ -757,7 +757,59 @@ If you use a virtualenv for your current project, add a path to it in your [proj
...
+                            "enabled": false,
+                            "live_mode": true,
+                        },
+                    }
+                }
+            }
+        },
+
+	}
+}
```
Comment (rchl) [inline docs/index.md:806]: Please fix indentation as it's inconsistent.
And remove empty line.

**Comment by ykqmain** [comment]
2 spaces confuses me, I'm very sorry, I closed it.

**Pull request closed without merging**